src/components/AttributesItems/DeclarativeAttributesItems.tsx

```diff
@@ -0,0 +1,40 @@
+import { Grid, GridItem } from '@patternfly/react-core';
+import { css } from '@patternfly/react-styles';
+import { PropsWithChildren } from 'react';
+
+import { EmptyStateSymbol } from 'components/EmptyStates/EmptyStateSymbol';
+import { TooltipWrapper } from 'components/TooltipWrapper/TooltipWrapper';
+
+import styles from './AttributesItems.module.css';
+
+interface IAttributesItemsProps {
+  isGridUniform?: boolean;
+}
+
+/**
+ * Represents a stylized name-value table component.
+ */
+export const AttributesItems = ({ children, isGridUniform = true }: PropsWithChildren<IAttributesItemsProps>) => (
```

Comment:
Personally I would suggest using `Attributes` and `AttributesItem`, so that it would be similar with PatternFly naming styles(e.g., `Flex`/`FlexItem`, `ToolBar`/`ToolBarItem`. This would be clearer than `AttributesItems`/`AttributesItem`, which looks too similar.

Reply:
naming - yes, you are right and we already discussed it with patrik (see my comments above), it's just temporary solution